### PR TITLE
Translation tab title

### PR DESCRIPTION
**Pull request by xueqingxiao**

### CODE REVIEW

File: components/tabs/demo/basic.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 order: 0
-title: 基本
+title: 
+  zh-CN: 基本
+  en-US: Basic.
```
Comment (afc163) [inline components/tabs/demo/basic.md]: No full stop in title.

File: components/tabs/demo/custom-add-trigger.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 order: 11
-title: 自定义新增页签触发器
+title: 
+  zh-CN: 自定义新增页签触发器
+  en-US: Customize trigger of new tab.
```
Comment (afc163) [inline components/tabs/demo/custom-add-trigger.md]: Customized

File: components/tabs/demo/card-top.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
 ---
 order: 10
-title: 卡片式页签容器
+title: 
+  zh-CN: 卡片式页签容器
+  en-US: Container of card type Tab.
```
Comment (afc163) [inline components/tabs/demo/card-top.md]: Just `Card type Tab`

**Comment by xueqingxiao** [comment]
@afc163 fix

**Comment by xueqingxiao** [comment]
@afc163 done!

**Comment by afc163** [comment]
Thx!

**Pull request merged**